Added `installApp` command to Appium doc (#1322)
saucelabs/sauce-docs: docs/mobile-apps/automated-testing/appium/real-devices.md

"appium:app","https://github.com/test-apps/ios-app.ipa"
```

> You can also install an app during a test by using `mobile: installApp` command. 

Reviewer: Maybe change it to this

You can also install a dependent app or an app upgrade during a test by using [](http://appium.io/docs/en/commands/device/app/install-app/) command. 

```java
driver.installApp("https://github.com/saucelabs/my-demo-app-rn/releases/download/v1.3.0/Android-MyDemoAppRN.apk");
```

> :::note Limitations

Reviewer: I would change the order and adjust it a bit to this maybe?

* The provided app path needs to be publicly available, this method does not have access to your local path/local storage
* This method does not have access to apps in our Sauce Storage. Only apps that are publicly available can be installed with this command.
* Because this method doesn't have access to our Sauce Storage we also can't re-sign, see below, and instrument the app. All instrumentation, see [App Settings](https://docs.saucelabs.com/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) options will not work for apps installed using `mobile: installApp` command.
* This method will not work by default for iOS due to signing. Each iOS app needs to be resigned so it is allowed to be installed on our devices. In order to make this work you must use a private device and add UDID of the private device to the provisioning profile for iOS.

"appium:app","https://github.com/test-apps/ios-app.ipa"
```

> You can also install a dependent app or an app upgrade during a test by using `driver.install('path-to-app')`command. 

Author: Done.

> You can also install a dependent app or an app upgrade during a test by using `driver.install('path-to-app')`command. 

Reviewer: ```suggestion
You can also install a dependent app or an app upgrade during a test by using `driver.installApp('path-to-app')`command. 
```

Author: Done :)

```js title=Driver App Example
driver.installApp("https://github.com/saucelabs/my-demo-app-rn/releases/download/v1.3.0/Android-MyDemoAppRN.apk");
```

:::note Limitations
* The provided app path needs to be publicly available as this method does not have access to your local path/storage.

> * This method does not have access to apps in our Sauce Storage. Only apps that are publicly available can be installed with this command. Therefore, we also can't re-sign and instrument the app. The Instrumentation will not work for apps installed using `driver.install('path-to-app')` command (check [App Settings](/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) options to learn more).

Reviewer: ```suggestion
* This method does not have access to apps in our Sauce Storage. Only apps that are publicly available can be installed with this command. Therefore, we also can't re-sign and instrument the app. The Instrumentation will not work for apps installed using `driver.installApp('path-to-app')` command (check [App Settings](/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) options to learn more).
```

> * This method will not work for iOS due to signing. To make it work, you must use a private device and add UDID of the private device to the provisioning profile for iOS.

Reviewer: Should we refer to a link in our docs that explains how signing works. The word might not always mean something to them without context

Author: Done :)

:::

> For more information about this command, please check the official [Appium website](http://appium.io/docs/en/commands/device/app/install-app/).

Reviewer 2: ```suggestion
For more information about this command, see the [Appium documentation](http://appium.io/docs/en/commands/device/app/install-app/).
```
